Cargo.toml

```diff
@@ -12,3 +12,7 @@ members = [
   "filecoin-hashers",
   "storage-proofs-update",
 ]
+
+[patch.crates-io]
+bellperson = { git = "https://github.com/filecoin-project/bellperson", branch = "forward-port-updates" }
+neptune = { git = "https://github.com/filecoin-project/neptune", branch = "updated-deps" }
```

Comment:
FIXME: Requires https://github.com/filecoin-project/bellperson/pull/274 and https://github.com/filecoin-project/neptune/pull/150 followed by crate releases